issuetracker/pivotaltracker/pivotaltracker.go

```diff
@@ -30,12 +30,12 @@ func (it *IssueTracker) TaskURLFrom(taskID string) string {
 // IssueAPIOrigin returns the URL for pivotaltracker's issue-fetching API
 func (it *IssueTracker) IssueAPIOrigin() string {
 	tokens := common.RemoveEmptyTokens(strings.Split(it.Origin, "/"))
-	if tokens[0] == "pivotaltracker.com" {
+	if strings.HasPrefix(tokens[0], "pivotaltracker.com") {
```

Comment:
This is actually not the same as the original implementation. We want the string to be exactly `pivotaltracker.com`, rather than having the prefix.

Please revert this change